hack/make-rules/test.sh

```diff
@@ -161,12 +161,12 @@ while getopts "hp:i:" opt ; do
       kube::test::usage
       exit 1
       ;;
-    ?)
+    :)
+      kube::log::usage "Option -${OPTARG} <value>"
       kube::test::usage
       exit 1
       ;;
-    :)
-      kube::log::usage "Option -${OPTARG} <value>"
+    ?)
```

Comment:
fix this lint failure
```
In ./hack/make-rules/test.sh line 164:
    ?)
    ^-- SC2221: This pattern always overrides a later one.

In ./hack/make-rules/test.sh line 168:
    :)
    ^-- SC2222: This pattern never matches because of a previous pattern.
```